tools/reviewer/web/src/app/dashboard/diff/diff-header/diff-header-content/diff-header-content.component.ts

```diff
@@ -0,0 +1,148 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { MatDialog, MatDialogRef } from '@angular/material';
+
+import { Diff, Reviewer } from '@/shared/proto';
+import {
+  AuthService,
+  FirebaseService,
+  NotificationService,
+} from '@/shared/services';
+import { AddUserDialogComponent } from '../add-user-dialog';
+import { DiffHeaderService } from '../diff-header.service';
+
+// The component implements content of the header
+// How it looks: "/src/assets/design-blocks/header-content.jpg"
```

Comment:
Let's not add these images to the repo. They just take up space. How about we host them in an image hosting service? (e.g imgur).

Reply:
It's 331.6 kB, but ok